rvgo/slow/vm.go

```diff
@@ -122,6 +122,12 @@ func Step(calldata []byte, po PreimageOracle) (stateHash common.Hash, outErr err
 	}
 
 	stateContentOffset := uint8(4 + 32 + 32 + 32 + 32)
```

Comment:
Can we make `stateContentOffset` a `uint16` by default? or is it used elsewhere as a `uint8` that I'm missing

Reply:
No other intent! I changed `stateContentOffset` to `uint16` by default :)